testing/src/main/java/io/stargate/it/http/docsapi/BaseDocumentApiV2Test.java

```diff
@@ -558,6 +568,27 @@ public void testPutReplacingWithArray() throws IOException {
     assertThat(OBJECT_MAPPER.readTree(r)).isEqualTo(wrapResponse(obj, "1", null));
   }
 
+  @Test
+  public void testPutProfile() throws IOException {
+    JsonNode obj =
+        OBJECT_MAPPER.readTree(this.getClass().getClassLoader().getResource("example.json"));
+    String resp = RestUtils.put(authToken, collectionPath + "/1?profile=true", obj.toString(), 200);
+    assertThat(OBJECT_MAPPER.readTree(resp).get("profile").isEmpty()).isFalse();
```

Comment:
Can we at least in one place assert on the structure of the profile JSON?

Reply:
This is done by unit tests. ITs here just make sure something is returned. I'd like to avoid duplicate tests for the same functionality and unit tests are more convenient for validating actual data.